squape/squishserver.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+import os
+
+from remotesystem import RemoteSystem
+
+from squape.internal.exceptions import SquishserverError
+from squape.report import log
+
+
+class SquishServer:
+    """Class to represent a local or remote squishserver"""
+
+    def __init__(self, location=None, host="127.0.0.1", port=4322):
+        """Open an RemoteSystem connection to a machine with a running squishserver
+
+        Args:
+            location (_type_, optional):    location of the Squish package.
+                                            Defaults to the "SQUISH_PREFIX".
+            host (str, optional): host of the squishserver. Defaults to "127.0.0.1".
+            port (int, optional): port of the squishserver. Defaults to 4322.
+        """
+        if location is None:
+            self.location = os.environ["SQUISH_PREFIX"]
```

Comment:
what if the variable is not set?

Reply:
If the environment variable "SQUISH_PREFIX" is not set, test script will throw the KeyError

Reply:
I am not sure if this should be SquishserverError, as it may not be a problem of the squishserver itself.